pytest_inmanta/test_parameter/boolean_parameter.py

```diff
@@ -51,7 +52,7 @@ def __init__(
         environment_variable: str,
         usage: str,
         *,
-        default=False,
+        default: Optional[Union[bool, DynamicDefault[bool]]] = False,
```

Comment:
Can you also update the signature of the `__init__` method for the other classes?

Reply:
done